client.yml

```diff
@@ -2863,6 +2863,130 @@ paths:
       name: id
       in: path
       required: true
+  /services/{id}/edge-routers:
+    get:
+      security:
+      - ztSession: []
+      - oauth2:
+        - openid
+      description: |
+        Retrieves a list of edge router resources that a specific service can dial or bind.
```

Comment:
that sounds weird to me
how about `list of edge router that can be used to dial/bind given service`?

Reply:
"Retrieves the list of edge routers authorized to dial or bind the specified service" ?

Reply:
sounds wrong because ERs are not dialing or binding, they are used to dial/bind by an identity

Reply:
yeah. probably can just remove the dial/bind thing entirely? "Retrieves the list of edge routers usable by the specified service" ?